assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -317,6 +331,21 @@ class GettingStartedOverview extends React.Component<Props, State> {
             />
           </Box>
 
+          <Box mb={3}>
+            <label htmlFor="greeting">
```

Comment:
this one still needs to be updated to `away_message` as well :) also mind adding a screenshot of the updated UI to the PR description @a8t?